### ISSUE TITLE
Can I get the hashes for the zip and ova ?

### ISSUE THREAD

**Issue by jmweila** [jzadeh/chiron-elk#8]

**Comment by jzadeh** [comment]
Sure ! Just to make sure you just want file checksums to validate the image with?   @rsfl can you post the latest  md5/checksum hash and link to the zip/ovas for chiron in this issue tracker or send them to me and I will post them.

**Comment by jmweila** [comment]
Yes -- the checksum to validate the downloads.  I have having issues and I
wanted to ensure that I had a good download before I asked a bunch of
questions.

Thanks!
